### DISCUSSION TITLE
Creating a domain for media type handlers in the exposed schema or not?

### DISCUSSION THREAD

**Discussion by christiaanwesterbeek** [PostgREST/postgrest#3751] [Q&A]
This way is documented in https://docs.postgrest.org/en/v12/how-tos/providing-images-for-img.html

```
create domain "application/octet-stream" as bytea;
```

But it seems more appropriate/organized when the domain is created in the same schema as the exposed schema.

```
create domain api."application/octet-stream" as bytea;
```

When would the latter be more appropriate?

**Comment by wolfgangwalther** [comment]
I'm always confused on where, i.e. in which schema, to add those domains. But apparently they work in any schema, IIRC.

I really don't have an opinion on where those domain types should be, I don't see any arguments for any specific schema.

"seems more appropriate" doesn't sell it to me, yet.
